### PR TITLE
Adding Sample notebooks to OSS

### PR DESCRIPTION
**Pull request by BhavanaTumma**

### CODE REVIEW

File: assets/models/system/bert-large-cased/description.md [hunk 1]
```diff
@@ -63,6 +63,13 @@ You can use the raw model for either masked language modeling or next sentence p
 
 Note that this model is primarily aimed at being fine-tuned on tasks that use the whole sentence (potentially masked) to make decisions, such as sequence classification, token classification or question answering. For tasks such as text generation you should look at model like GPT2.
 
+# Inference samples
+
+Inference type|Python sample (Notebook)
+|--|--|
+Real time|[sdk-example.ipynb](https://aka.ms/sdk-notebook-examples)
```
Comment (novaturient95) [inline assets/models/system/bert-large-cased/description.md:70]: all notebooks needed?

File: assets/models/system/bert-large-cased/description.md [hunk 2]
```diff
@@ -63,6 +63,13 @@ You can use the raw model for either masked language modeling or next sentence p
...
+Real time|[fill-mask-online-endpoint.ipynb](https://aka.ms/fill-mask-online-endpoint-oss)
```
Comment (novaturient95) [inline assets/models/system/bert-large-cased/description.md:71]: Could we add batch as well?

Comment (novaturient95) [inline assets/models/system/bert-large-cased/description.md:71]: we can follow evaluation and FT sample example and add datasets and use case as well, so all samples are consistent

**Review by skasturi — APPROVED** [review]

**Review by novaturient95 — APPROVED** [review]

File: assets/models/system/bert-large-cased/description.md (hunk 2 above)
Comment (novaturient95) [reply assets/models/system/bert-large-cased/description.md:71]: TODO in next PRs if needed

Comment (novaturient95) [reply assets/models/system/bert-large-cased/description.md:71]: TODO in next PRs if needed

File: assets/models/system/bert-large-cased/description.md (hunk 1 above)
Comment (novaturient95) [reply assets/models/system/bert-large-cased/description.md:70]: TODO in next PRs if needed

**Pull request merged**